### PR TITLE
Rx

### PR DESCRIPTION
**Pull request by ArqTras**
to check src\crypt\rx_slow-hash

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -23,7 +23,7 @@ set(ARQMA_SOURCE_DIR ${ARQMA_DIR}
         CACHE PATH "Path to the root directory for Arqma")
 
 # set location of arqma build tree
-set(ARQMA_BUILD_DIR ${ARQMA_SOURCE_DIR}/build/Linux/release-v0.6.0/release
+set(ARQMA_BUILD_DIR ${ARQMA_SOURCE_DIR}/build/release/
```
Comment (malbit) [inline CMakeLists.txt]: If you add “/“ at the end output “${ARQMA_SOURCE_DIR}/something” will look:
“~/arqma//something”

File: cmake/FindArqma.cmake [hunk 2]
```diff
@@ -44,7 +44,7 @@ foreach (l ${LIBS})
 	find_library(Arqma_${L}_LIBRARY
 		NAMES ${l}
 		PATHS ${CMAKE_LIBRARY_PATH}
-		PATH_SUFFIXES "/src/${l}" "/src/" "/external/db_drivers/lib${l}" "/lib" "/src/crypto" "/contrib/epee/src" "/external/easylogging++/" "/external/${l}"
+		PATH_SUFFIXES "/src/${l}" "/src/" "/external/db_drivers/lib${l}" "/lib" "/src/crypto" "/contrib/epee/src" "/external/easylogging++/" "/external/randomarq"
```
Comment (malbit) [inline cmake/FindArqma.cmake]: ${l} means all libraries and dont need to point exact one

File: cmake/FindArqma.cmake [hunk 3]
```diff
@@ -64,21 +64,15 @@ if (EXISTS ${ARQMA_BUILD_DIR}/src/ringct/libringct_basic.a)
 			PROPERTY IMPORTED_LOCATION ${ARQMA_BUILD_DIR}/src/ringct/libringct_basic.a)
 endif()
 
-if(EXISTS ${ARQMA_BUILD_DIR}/external/randomarq/librandomx.a)
-  message(STATUS FindArqma " found librandomx.a")
-  add_library(randomx STATIC IMPORTED)
-  set_property(TARGET randomx PROPERTY IMPORTED_LOCATION ${ARQMA_BUILD_DIR}/external/randomarq/librandomx.a)
-endif()
```
Comment (malbit) [inline cmake/FindArqma.cmake:71]: Why did you delete that?

File: src/crypto/rx-slow-hash.c [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+
+void me_rx_slow_hash_allocate_state(void) {
+}
+
+void me_rx_slow_hash_free_state(void) {
+  if (rx_vm != NULL) {
+    randomx_destroy_vm(rx_vm);
+    rx_vm = NULL;
+  }
+}
```
Comment (malbit) [inline src/crypto/rx-slow-hash.c]: That rx-slow-hash.c does not match ours :)

**Review by malbit — CHANGES_REQUESTED** [review]

File: cmake/FindArqma.cmake (hunk 2 above)
Comment (ArqTras) [reply cmake/FindArqma.cmake]: In that case easylogging++ can be removed as well as randomarq 
(it's not issue)

File: CMakeLists.txt (hunk 1 above)
Comment (ArqTras) [reply CMakeLists.txt]: right

File: src/crypto/rx-slow-hash.c (hunk 4 above)
Comment (ArqTras) [reply src/crypto/rx-slow-hash.c]: removed

File: cmake/FindArqma.cmake (hunk 3 above)
Comment (ArqTras) [reply cmake/FindArqma.cmake:71]: added
